lib/stackConfiguration/src/main/resources/stack.properties

```diff
@@ -339,3 +337,5 @@ org.sagebionetworks.hmac.signing.key.current.version=0
 # The maximum number of months to process for monthly statistics
 org.sagebionetworks.statistics.monthly.max=12
 
+# Whether the trash can purge is enabled or not
```

Comment:
Can user the semaphore locks in the DB to lock out the trash worker, then release the lock when ready to resume.